### ISSUE TITLE
Missing node types ( after updating comfyUI )

### ISSUE THREAD

**Issue by kulayumi** [kijai/ComfyUI-CogVideoXWrapper#293]
Thank you for your contribution!

I just wanted to try the "LTXV video" program today, so I upgraded comfyUI. Later, when I used CogVideoXWrapper, I found that the node could not be used, and it could not be used even if the node was updated or reinstalled. What steps did I miss ? Thank you !

![20241125](https://github.com/user-attachments/assets/52067749-2c9c-4a3a-b8b6-4c0aceee2f3a)

FETCH DATA from: F:\ComfyUI_windows_portable test\ComfyUI\custom_nodes\ComfyUI-Manager-main\extension-node-map.json [DONE]
[Inspire Pack] IPAdapterPlus is not installed.
Error. No naistyles.csv found. Put your naistyles.csv in the custom_nodes/ComfyUI_NAI-mod/CSV directory of ComfyUI. Then press "Refresh".
                  Your current root directory is: F:\ComfyUI_windows_portable test\ComfyUI

Error. No naistyles.csv found. Put your naistyles.csv in the custom_nodes/ComfyUI_NAI-mod/CSV directory of ComfyUI. Then press "Refresh".
                  Your current root directory is: F:\ComfyUI_windows_portable test\ComfyUI

Error. No naistyles.csv found. Put your naistyles.csv in the custom_nodes/ComfyUI_NAI-mod/CSV directory of ComfyUI. Then press "Refresh".
                  Your current root directory is: F:\ComfyUI_windows_portable test\ComfyUI

Error. No naistyles.csv found. Put your naistyles.csv in the custom_nodes/ComfyUI_NAI-mod/CSV directory of ComfyUI. Then press "Refresh".
                  Your current root directory is: F:\ComfyUI_windows_portable test\ComfyUI

Error. No naistyles.csv found. Put your naistyles.csv in the custom_nodes/ComfyUI_NAI-mod/CSV directory of ComfyUI. Then press "Refresh".
                  Your current root directory is: F:\ComfyUI_windows_portable test\ComfyUI

Error. No naistyles.csv found. Put your naistyles.csv in the custom_nodes/ComfyUI_NAI-mod/CSV directory of ComfyUI. Then press "Refresh".
                  Your current root directory is: F:\ComfyUI_windows_portable test\ComfyUI
                  
                   

** Platform: Windows
** Python version: 3.11.8 (tags/v3.11.8:db85d51, Feb  6 2024, 22:03:32) [MSC v.1937 64 bit (AMD64)]
** Python executable: F:\ComfyUI_windows_portable test\python_embeded\python.exe
** Log path: F:\ComfyUI_windows_portable test\comfyui.log

Total VRAM 12282 MB, total RAM 48996 MB
pytorch version: 2.4.0+cu121
F:\ComfyUI_windows_portable test\python_embeded\Lib\site-packages\xformers\ops\fmha\flash.py:211: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.
  @torch.library.impl_abstract("xformers_flash::flash_fwd")
F:\ComfyUI_windows_portable test\python_embeded\Lib\site-packages\xformers\ops\fmha\flash.py:344: FutureWarning: `torch.library.impl_abstract` was renamed to `torch.library.register_fake`. Please use that instead; we will remove `torch.library.impl_abstract` in a future version of PyTorch.
  @torch.library.impl_abstract("xformers_flash::flash_bwd")
xformers version: 0.0.27.post2
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4070 SUPER : cudaMallocAsync
Using xformers cross attention

**Comment by kijai** [comment]
None of those errors have anything to do with these nodes, but if you installed the LTXV custom nodes, they (quite annoyingly) force old version of diffusers, which won't work with these nodes. Re-installing these nodes should fix it, or manually updating diffusers to the latest version (0.31.0)

**Comment by Ainaemaet** [comment]
@kijai yes, it is annoying that they do that - I have to manually pip install req for one of your wrappers each time I upgrade LTXV XD

**Comment by kijai** [comment]
> @kijai yes, it is annoying that they do that - I have to manually pip install req for one of your wrappers each time I upgrade LTXV XD

They have fixed that now after I complained about it, was just an oversight.
